Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -108,20 +76,30 @@ static float AspectToWidescreen(float aspect)
   return aspect * ((16.0f / 9.0f) / (4.0f / 3.0f));
 }
 
-Renderer::Renderer()
+Renderer::Renderer(int backbuffer_width, int backbuffer_height)
 {
+  FramebufferManagerBase::SetLastXfbWidth(MAX_XFB_WIDTH);
+  FramebufferManagerBase::SetLastXfbHeight(MAX_XFB_HEIGHT);
+
   UpdateActiveConfig();
 
+  m_backbuffer_width = backbuffer_width;
```

Comment:
width and height can likely be set in the constructor initializer list, assuming none of the preceding function calls do weird shenanigans with the class variables (if they do, a comment here to explain that might be helpful).